### DISCUSSION TITLE
Any way around the EFI requirement?

### DISCUSSION THREAD

**Discussion by mgrusin** [Sithuk/ubuntu-server-zfsbootmenu#28] [Q&A]
I'm installing on a very old system (late 2000s supermicro) and the script fails with "boot environment check failed. EFI boot environment not found. script requires EFI." Any suggestions appreciated, thank you!

**Comment by Sithuk** [comment]
Hi mgrusin,

The script doesn't currently support BIOS systems. However, the good news is that zfsbootmenu does. The zbm authors recommend using syslinux instead of rEFInd for a BIOS system. The syslinux install documentation can be found at the link below.

https://docs.zfsbootmenu.org/en/latest/guides/void-linux/syslinux-mbr.html

**Comment by Sithuk** [comment]
You could also check out the following ubuntu install script.
https://github.com/prabirshrestha/simple-ubuntu-installer
